tests/test_integration_fairflow.py

```diff
@@ -1169,7 +1169,7 @@ def test_integration_group_not_found_error(
         content_type="application/json",
         headers=openreview_client.headers,
     )
-    assert response.status_code == 200
+    assert response.status_code == 404
```

Comment:
why do you need this change here?

Reply:
The match_group validity runs during the init call of interface, which fails and thus the post call fails as well.

Reply:
I think I would prefer to run it inside of the job, I need to check how it looks the UI when there is an error clicking the 'Run Match' button.